### DISCUSSION TITLE
join testnet problems

### DISCUSSION THREAD

**Discussion by xcfyl** [filecoin-project/lotus#6985] [Storage Provider] [answered]
When I try to join the calibnet，I got the warning: other peer has different genesis.How can I solve this problems?

**Comment by de-authority** [comment, marked as answer]
what's you lotus version. I don't think you run the right version. can you share more detail.

**Reply by xcfyl** [reply]
i use git checkout v1.10.0-rc6, beacause this link shows https://network.filecoin.io/#calibration which tag I should use.

**Reply by de-authority** [reply]
yeah. i saw that .can you sync to the chain?

**Reply by xcfyl** [reply]
i can't

**Reply by xcfyl** [reply]
if I don't use this branch,which branch should I use in test net?

**Reply by dd45e640b42e6da7da96faee3996ef7c** [reply]
the latest release branch, https://github.com/filecoin-project/lotus/releases/tag/v1.11.0

(all releases: https://github.com/filecoin-project/lotus/tags)

then run `make calibnet` or `make nerpanet` depending on what testnet you want to join

https://docs.filecoin.io/get-started/lotus/installation/#build-and-install-lotus

![2021-08-05_19-19](https://user-images.githubusercontent.com/60559388/128341786-8e36bc78-535d-4759-ac1d-5d0fd43d8758.png)

**Reply by xcfyl** [reply]
thank for your answer,but I also has a question.I exec the cmd:
1、git clone https://github.com/filecoin-project/lotus.git
2、cd lotus/
3、make clean calibnet
4、lotus daemon
when the chain sync completed,I create the wallet address,I found my wallet addr has preffix f not the t.
This means I joined the mainnet not the test net,I don't know why

**Comment by LexLuthr** [comment]
@dingqinglei Were you able to join calibnet? if yes, then can you tell us how you resolved this?
